compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -596,26 +596,20 @@ public void visit(BLangConstant astConstant) {
         this.env.enclPkg.constants.add(birConstant);
     }
 
+    private ConstValue getBIRConstantVal(BLangConstantValue constValue, BType type) {
```

Comment:
Why do we need a separate method? Instead of updating the existing method to accept the type?